model/src/main/kotlin/VcsInfo.kt

```diff
@@ -100,6 +100,11 @@ data class VcsInfo(
     fun toCuration() = VcsInfoCurationData(type, url, revision, path)
 }
 
+/**
+ * Return this [VcsInfo] if not null or else [VcsInfo.EMPTY].
+ */
+fun VcsInfo?.orEmpty(): VcsInfo = this ?: VcsInfo.EMPTY
```

Comment:
I'm myself a bit undecided, but shouldn't this strictly speaking be put into `model/src/main/kotlin/utils/Extensions.kt`?

Reply:
I prefer to keep it here but wouldn't mind moving it if you want me to. Note that we have similar extension e.g. in `LicenseClassifications.kt`, `Resolutions.kt`.